cmd/test_cedar_intake/main.go

```diff
@@ -437,12 +448,43 @@ var rootCmd = &cobra.Command{
 	Long:  `Utility for either submitting test data to the CEDAR Intake API or dumping the JSON payload that would be submitted to a local file`,
 }
 
-func init() {
-	rootCmd.AddCommand(submitCmd)
-	rootCmd.AddCommand(dumpCmd)
-}
-
 func execute() {
+	ctx := context.Background()
+	logger, err := zap.NewDevelopment()
+	if err != nil {
+		panic(fmt.Errorf("problem initializing dev logger in test_cedar_intake: %w", err))
+	}
+
+	ctx = appcontext.WithLogger(ctx, logger)
+
+	config := testhelpers.NewConfig()
+	dbConfig := storage.DBConfig{
+		Host:           config.GetString(appconfig.DBHostConfigKey),
+		Port:           config.GetString(appconfig.DBPortConfigKey),
+		Database:       config.GetString(appconfig.DBNameConfigKey),
+		Username:       config.GetString(appconfig.DBUsernameConfigKey),
+		Password:       config.GetString(appconfig.DBPasswordConfigKey),
+		SSLMode:        config.GetString(appconfig.DBSSLModeConfigKey),
+		MaxConnections: config.GetInt(appconfig.DBMaxConnections),
+	}
+
+	ldClient, err := ld.MakeCustomClient("fake", ld.Config{Offline: true}, 0)
+	if err != nil {
+		panic(fmt.Errorf("problem creating custom launchDarkly client in test_cedar_intake: %w", err))
+	}
+
+	store, err := storage.NewStore(dbConfig, ldClient)
+	if err != nil {
+		panic(fmt.Errorf("problem intializing store in test_cedar_intake: %w", err))
+	}
```

Comment:
Is this all necessary to create a context with dataloaders?

Reply:
It appears so - we could probably hide a chunk of it by creating `testhelpers.NewStore()` and doing all of the config stuff in there. But this stuff is necessary for dataloaders.

Reply:
@mynar7 btw, I was incorrect about ^ that comment. `package storage` imports  `testhelpers`, so `testhelpers` cannot create a store